metadata-ingestion/src/datahub/ingestion/source/unity/proxy_types.py

```diff
@@ -115,7 +122,7 @@ class ServicePrincipal:
 
 @dataclass(frozen=True, order=True)
 class TableReference:
-    metastore_id: str
```

Comment:
Thought this was confusing too, because `metastore` (name) and `metastore_id` are different things